utils.ts

```diff
@@ -9,26 +9,60 @@ import {join} from 'path';
 import {execSync} from 'child_process';
 import * as mkdirp from 'mkdirp';
 
-export function defaults<T>(val: T | undefined, def: T): T {
+export function defaults<T>(val: T|undefined, def: T): T {
   return typeof val === 'undefined' ? def : val;
 }
 
 export async function delay(time: number) {
   return new Promise(resolve => setTimeout(resolve, time));
 }
 
+export function getPlatform() {
+  const platform = process.platform;
+
+  if (platform !== 'linux') {
+    return platform;
+  }
+  // Windows Subsystem for Linux, reports Linux as platform but chrome is on Windows
+  // we need to find out the real platform
+  if (process.env.PATH && (process.env.PATH as string).includes(':/mnt/c/Windows/System32:')) {
```

Comment:
this feels a little brittle.. (i forget if a windows install on non-C-drive is possible, but...)

I saw some other projects [that basically do](https://github.com/kairess/projectETTN/blob/8bdfbbe80f7330d92de3f8481c4fd8f99128262e/.meteor/local/build/programs/server/mini-files.js#L16-L22) `require('os').release().includes('Microsoft')`

but it looks like even that is incomplete: https://github.com/sindresorhus/is-wsl/commit/93b4fcb68f3a955335266c5757317ab680e9ff8f

maybe let's pull in is-wsl.

Reply:
Awesome, didn't know about it. I'm doing a `require` because of:
> file: 'file:///c%3A/Projects/chrome-launcher/utils.ts'
severity: 'Error'
message: 'Could not find a declaration file for module 'is-wsl'. 'c:/Projects/chrome-launcher/node_modules/is-wsl/index.js' implicitly has an 'any' type.
  Try `npm install @types/is-wsl` if it exists or add a new declaration (.d.ts) file containing `declare module 'is-wsl';`'
at: '11,24'
source: 'ts'

I could add the `.d.ts.` file but wasn't sure what you prefer.